content/en/docs/concepts/overview/components.md

```diff
@@ -125,7 +125,12 @@ saving container logs to a central log store with search/browsing interface.
 
 ## {{% heading "whatsnext" %}}
 
-* Learn about [Nodes](/docs/concepts/architecture/nodes/)
-* Learn about [Controllers](/docs/concepts/architecture/controller/)
-* Learn about [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/)
-* Read etcd's official [documentation](https://etcd.io/docs/)
+Learn more about the following:
+   * [Nodes](/docs/concepts/architecture/nodes/) and [communication between Nodes and control plane](/docs/concepts/architecture/control-plane-node-communication/).
+  
+   * [Controllers](/docs/concepts/architecture/controller/) in Kubernetes.
+   * [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) is the default scheduler for Kubernetes and runs as part of the control plane.
+   * Read etcd's official [documentation](https://etcd.io/docs/).
+   * Several common [container runtimes](/docs/setup/production-environment/container-runtimes/) in Kubernetes.
+   * Integrating with cloud providers using [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/).
+   * [kubectl](/docs/reference/generated/kubectl/kubectl-commands) commands.
```

Comment:
The indentations for the above list items do not make sense.

The revise paragraph doesn't flow well ... some items are phrases, some items are sentences ...

Reply:
Hi @tengqm,
Thanks for your review comment,
How about this?

Learn more about the following:
- [Nodes](/docs/concepts/architecture/nodes/) and [their communication](/docs/concepts/architecture/control-plane-node-communication/) with the control plane.
-  Kubernetes [controllers](/docs/concepts/architecture/controller/).
-  [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) which is the default scheduler for Kubernetes.
- Etcd’s official [documentation](https://etcd.io/docs/).
- Several [container runtimes](/docs/setup/production-environment/container-runtimes/) in kubernetes.
- Integrating with cloud providers using [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/).
- [kubectl](/docs/reference/generated/kubectl/kubectl-commands) commands.

Reply:
Hi @tengqm, @sftim, 
could you please take a look on above preview text and share your feedback.

Reply:
Hi @tengqm,
Updated as per the suggested  feedback, could you please take an another look.